.kitchen.docker.yml

```diff
@@ -0,0 +1,62 @@
+---
+driver:
+  name: dokken
+  chef_version: latest
+
+transport:
+  name: dokken
+
+verifier:
+  root_path: '/opt/verifier'
+  sudo: false
+
+provisioner:
+  name: dokken
+
+platforms:
+  - name: centos-6.8
+    driver:
+      image: centos:6.8
+      pid_one_command: /sbin/init
+      intermediate_instructions:
+        - RUN /usr/bin/yum install -y sudo
```

Comment:
We solved this by using a specified version of chef. 12.15.19 as you also suggested.